trajectory_tracker/src/trajectory_tracker.cpp

```diff
@@ -460,8 +462,9 @@ void TrackerNode::control()
       wref = std::copysign(1.0, wref) * vel_[1];
     }
 
+    const double k_ang = (gain_at_vel_ == 0.0) ? (k_[1]) : (k_[1] * linear_vel / gain_at_vel_);
     w_lim_.increment(
-        dt * (-dist_err_clip * k_[0] - angle * k_[1] - (w_lim_.get() - wref) * k_[2]),
+        dt * (-dist_err_clip * k_[0] - angle * k_ang - (w_lim_.get() - wref) * k_[2]),
```

Comment:
![image](https://user-images.githubusercontent.com/8390204/51472675-7de9a780-1dbd-11e9-824f-128ea2707090.png)
Decreasing both `K_{phi}` and `K_{omega}` let system unstable maybe due to the non-linearity of clipping.
Decreasing only `K_{phi}` works as expected.